go.mod

```diff
@@ -52,7 +52,7 @@ require (
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/pelletier/go-toml v1.6.0 // indirect
 	github.com/pierrec/lz4 v2.4.1+incompatible // indirect
-	github.com/pkg/errors v0.9.1 // indirect
+	github.com/pkg/errors v0.9.1
```

Comment:
we're no longer using this package, errors can be wrapped via `fmt.Errorf("... : %w", err)`